spec/text-objects-spec.coffee

```diff
@@ -164,28 +164,232 @@ describe "TextObjects", ->
       expect(editorElement.classList.contains('normal-mode')).toBe(true)
 
   describe "the 'it' text object", ->
+
     beforeEach ->
-      editor.setText("<something>here</something><again>")
-      editor.setCursorScreenPosition([0, 5])
+      editor.setText('clean')
+      editor.setCursorBufferPosition([0, 0])
+      keydown('y')
+      keydown('i')
+      keydown('w')
+      expect(vimState.getRegister('"').text).toBe('clean')
```

Comment:
Lol, I understand the confusion. I just wanted to make sure that I reset the register before each test, and that was the first thing I came up with. The plan was to find a better way to set the value in the register, but I obviously forgot :)